.travis.yml

```diff
@@ -40,6 +40,7 @@ script:
   - './cc-test-reporter sum-coverage coverage/codeclimate.*.json'
   - './cc-test-reporter upload-coverage'
   - 'bundle exec bundle-audit check --update'
+  - yarn build-storybook
```

Comment:
We just want to ensure that this builds. If it doesn't something is really broken in the UI or stories were not updated.